### ISSUE TITLE
Enums are received as string, while the generated enum types them as number

### ISSUE THREAD

**Issue by micmro** [kondi/rxjs-grpc#23]
I have a proto-file containing a enum, which get's interpreted as an enum of numbers, however when sending down an enum (as number) from the server the client receives it as string.

### Setup 
I've create a runnable example, based on your helloworld in [my fork](https://github.com/micmro/rxjs-grpc/tree/enum-bug-example/examples/helloworld):
You can see the adjustments here: https://github.com/kondi/rxjs-grpc/compare/master...micmro:enum-bug-example

### Behaviour 
Input value on the server: `Room.SECOND` (`Room.SECOND === 1`)
Output of `npm run client`:
```
Greeting: Hello world
Multi greeting: Hello world, SECOND (string)
Multi greeting: Hello world, SECOND (string)
Multi greeting: Hello world, SECOND (string)
```

Expected output based on types:
```
Greeting: Hello world
Multi greeting: Hello world, 1 (number)
Multi greeting: Hello world, 1 (number)
Multi greeting: Hello world, 1 (number)
```

 ### Thoughts
Could this be related to an upstream protobuf.js (e.g. [this issue](https://github.com/dcodeIO/protobuf.js/issues/918))?

**Comment by micmro** [comment]
Looks like it's correct that it's the key being send, just the typing is wrong then:

> proto3 | JSON | JSON example | Notes
> -- | -- | -- | --
> enum | string | "FOO_BAR" | The name of the enum value as specified in proto is used.
https://developers.google.com/protocol-buffers/docs/proto3#json

**Comment by kondi** [comment]
If I remember well, I had the same issue, but I just made a custom fix for it at that time... This was the related issue if we talk about the same: https://github.com/grpc/grpc/issues/8726

**Comment by kondi** [comment]
Released in [v0.2.0](https://github.com/kondi/rxjs-grpc/releases/tag/v0.2.0).
